hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/pipeline/Pipeline.java

```diff
@@ -339,7 +339,7 @@ public Pipeline build() {
             nodeIndex--;
           }
         }
-        LOG.info("Deserialize nodesInOrder {} in pipeline {}", nodesWithOrder,
+        LOG.debug("Deserialize nodesInOrder {} in pipeline {}", nodesWithOrder,
```

Comment:
Can we change the LOG.info at line 195 to LOG.debug as well?

Reply:
Sure, np.